### ISSUE TITLE
[Bug]: Top Contributor Name is Not Visible in Dark Mode

### ISSUE THREAD

**Issue by Sindhuu-B** [sayeeg-11/OpenSource-Compass#786]
### Bug Description

In the Contributors page beside Top Contributor text the name of the top contributor is not visible in Dark Mode due to color contrast issues. Making it not visible to users

### Steps to Reproduce

1. Go to Footer
2. Click on Contributors
3. See the Top Contributor Title
4. The name of the top contributor is not visible In Dark Mode

### Expected Behavior

The name of the Top contributor should be clearly visible in Dark Mode

### Actual Behavior

The Top contributor name is not clearly visible in Dark Mode

### Browser

Chrome

### Operating System

Windows 11

## Dark Mode

<img width="1874" height="363" alt="Image" src="https://github.com/user-attachments/assets/4575769c-2324-4179-a103-b4ce7cd6efc3" />

## Light Mode
<img width="1848" height="324" alt="Image" src="https://github.com/user-attachments/assets/7d905300-6d75-4ab0-8eb5-4dd953d0d6b8" />

I am SWOC'26 Participant and would be happy to take this.

**Comment by Aarush2112** [comment]
Hello @sayeeg-11 , assign this issue to me under SWoC.

**Comment by sayeeg-11** [comment]
@Sindhuu-B  assigned
